lib/whimsy/asf/documents.rb

```diff
@@ -113,6 +113,7 @@ def self.find(name)
       result = nil
       if files
         stem = Regexp.new Regexp.quote name.downcase.gsub(' ','-')
+          .gsub(".", "").gsub(",", "")
```

Comment:
Why not use .gsub(%r{[,.]},"") ?
Also, maybe it would be better to replace all characters apart from the expected ones (alphanumeric, _,-).
It's generally safer to specify what is allowed, rather than what is disallowed, as it is easy to miss something.

Reply:
Also, given that this function searches a directory listing for matches, any name massaging should agree with whatever name massaging was used to create the files initially. There should be a single function to do this processing.

Reply:
It's just occurred to be that names can contain single-quotes.
Also non-ASCII letters

Reply:
done with your changes...